acclimatise/cli_types.py

```diff
@@ -79,7 +79,10 @@ class CliDir(CliType):
     Takes a directory path
     """
 
-    pass
+    output: bool = None
```

Comment:
Still would appreciate some subclassing here (and below)

Reply:
OK. Then I go for subclasses.

Reply:
Just add an CliOutputFile class? Or subclass CliInputFile and CliOutputFile from CliFile? Any preferences? I would go for the former option.

Reply:
Oh, that's not quite what I meant. I more meant that you add like a `CliTyp → CliFileSystemType → CliFile, CliDirectory`, whereby you put the `output` field on the abstract class `CliFileSystemType`. Just to avoid defining `output` identically in two places.